### PR TITLE
Drum Kit

### PR DESCRIPTION
**Pull request by hauteballe**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+.keys {
```
Comment (tibmt) [inline style.css:14]: I think `keys-wrapper` is more appropriate name

File: style.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  padding: 1rem 0.5rem;
+  transition: all 0.07s ease;
+  width: 10rem;
+  text-align: center;
+  color: white;
+  background: rgba(0, 0, 0, 0.4);
+  text-shadow: 0 0 0.5rem black;
+}
+
+.playing {
```
Comment (tibmt) [inline style.css:36]: Optional: I propose `playback` class name

File: main.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+function playSound(event) {
```
Comment (tibmt) [inline main.js:1]: Make `App` function, like in other your PR, to wrap your functionality in separate function

File: style.css (hunk 1 above)
Comment (hauteballe) [reply style.css:14]: renamed

File: main.js (hunk 3 above)
Comment (hauteballe) [reply main.js:1]: Done

**Pull request merged**